study.md

```diff
@@ -136,12 +155,17 @@ Ruby:
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
 # your answer here
+array.select { |num| num.odd? }
 ```
 
 JavaScript:
-
+not sure if I did this right
 ```javascript
-const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
-
+const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+const oddNum = array.filter(function(array){
+  for (let i = 0; i < array.length; i++) {
+    return array[i] % 2 !== 1
+  }
+})
 // your answer here
```

Comment:
so in JS, the callback function for filter doesn't take array as an argument. That callback function has the individual array element and the index.
`array.filter(function(element, i){
     return element % 2 !== 1;
})`